docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -249,6 +249,16 @@ kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/master/Documen
 ```
 {% endcapture %}
 
+{% capture kube-router %}
+
+Kube-router relies on kube-controll-manager to allocate pod CIDR for the nodes. So you must use kubeadm init with --pod-network-cidr flag. 
+
+Kube-router provides pod networking, network policy and high perfoming IPVS/LVS based service proxy as well. 
+
+Please follow the instruction [here](https://github.com/cloudnativelabs/kube-router/blob/master/Documentation/kubeadm.md) to setup Kubernetes cluster with Kube-router using kubeadm.
```

Comment:
For information on setting up Kubernetes cluster with Kube-router using kubeadm, see [kubeadm documentation](https://github.com/cloudnativelabs/kube-router/blob/master/Documentation/kubeadm.md).

Reply:
@radhikapc thanks. have incorporated changes.